README.md

```diff
@@ -67,3 +67,4 @@ Incluyan en [esta carpeta](/design) su propuesta de diseño
 - Colocar variables de entorno correspondientes copiando .env.example en un nuevo archivo .env
 - Ejecutar rails db:create
 - Si no es la primera vez asegurarse de realizar las migraciones con: rails db:migrate
+- Si se agrega un nuevo modelo ejecutar annotate para anotar los campos que conforman esa tabla en la parte superior del modelo
```

Comment:
We could avoid having to remember this extra step by automating this process 😄 
https://github.com/ctran/annotate_models#configuration-in-rails

Reply:
I agree with what @EduardoGHdez  mentions, likewise, I would just like to suggest that if you are indicating that a library be executed, you can put the command to execute.

Reply:
The command is annotate, its mentioned in the comment

Reply:
@EduardoGHdez  as you mention after this comment i already add the tasks for these actions, I've made an amend removing the  unnecessary comment at the readme file. Thanks for your CR.